public/stylesheets/sass/Navbar.scss

```diff
@@ -1,67 +1,187 @@
+		display: block;
+	}
+
+	@include smallMobile {
+		@include size($width: 5%);
+		display: block;
+	}
+}
+
+.LogoImage {
+	@include size($width: 37%,$height: 50%);
+	background-image: url("/images/logo.png");
+	background-repeat: no-repeat;
+	border-radius: 0;
+	background-size: 100%;
+	@include tablet {
+		@include size($width: 45%, $height: 48%);
+		@include flex($justify: center);
+	}
+
+	@include mobile {
+		@include size($width: 52%, $height: 34%);
+		@include flex($justify: center);
+	}
+
+	@include smallMobile {
+		@include size($width: 67%);
+		@include flex($justify: center);
+	}
+}
+
+.NavLinks {
+	@include size($width: 50%, $min-height: 100%);
+	@include flex($justify:flex-end, $align:center, $direction: row);
+	list-style: flex;
 	margin: 0;
-	padding-right: 10%;
-	list-style: none;
 	color: $white-color;
 
 	@include tablet {
-		@include flex($justify: space-around);
-		padding-right: 5%;
+		@include size($width: 90%, $min-height: 20%);
+		display: none;
+		justify-content: center;
+		flex-direction: column;
+		font-size: $large-font-size;
+	}
+
+	@include mobile {
+		@include size($width: 90%, $min-height: 20%);
+		display: none;
+		flex-direction: column;
+		justify-content: center;
+		font-size: $large-font-size;
+	}
+
+	@include smallMobile {
+		@include size($width: 100%, $min-height: 20%);
+		display: none;
+		flex-direction: column;
+		justify-content: center;
+		font-size: $large-font-size;
+		padding: 0;
+	}
+	a {
+		text-decoration: none;
+		font-size: $medium-font-size;
+		color: $white-font-color;
+		text-align: center;
+
+		@include tablet {
+			font-size: $small-font-size;
+		}
+
+		@include smallMobile {
+			font-size: $small-font-size;
+		}
+		&:hover{
+			text-decoration: none;
+			color: $white-font-color;
+		}
+	}
+
+	&.is-open {
+		display: flex;
 	}
 }
+
 .Link{
 	@include size($width:24%, $height:60%);
 	@include flex($align: center, $justify: center);
 	text-decoration: none;
 	flex-wrap: wrap;
 	color: $white-color;
+	list-style: none;
 
-}
-.Link:hover{
-	background-color: rgba(0,0,0,0.7);
-	text-decoration: none;
-	color: #34495e;
-	font-size: 1.1em;
-}
-.Link:hover a{
-	background-color: rgba(0,0,0,0.7);
-	color: $white-font-color;
-	font-size: 1.1em;
-}
 
-.NavLinks a{
-	text-decoration: none;
-	font-size: 1.1em;
-	color: $white-font-color;
-	text-align: center;
-}
-.NavLinks a:hover{
-	text-decoration: none;
-	color: $white-font-color;
+	@include tablet {
+		@include size($width: 90%, $height: 10vh);
+		border-top: 1px solid rgba(0,0,0,0.4);
+	}
+
+	@include mobile {
+		@include size($width: 90%, $height: 10vh);
+		border-top: 1px solid rgba(0,0,0,0.4);
+	}
+
+	@include smallMobile {
+		@include size($width: 90%, $height: 10vh);
+		border-top: 1px solid rgba(0,0,0,0.4);
+		font-size: $medium-font-size;
+	}
+	&:hover {
+		background-color: rgba(0,0,0,0.7);
+		text-decoration: none;
+		color: rgba(52, 73, 94,1);
+		font-size: $medium-font-size;
+	}
+	&:hover a {
+		background-color: rgba(0,0,0,0.7);
```

Comment:
Se a cor do hover for sempre esta ou for utilizada varias vezes pode ser criada uma variavel para ela, se acharem proveitoso.